content/handbook/product/product-management/learning-and-development.md

```
@@ -0,0 +1,710 @@
+- [John Cutler](https://twitter.com/johncutlefish)
+- [Ken Norton](https://twitter.com/kennethn)
+- [Jeff Patton](https://twitter.com/jeffpatton)
+- [Roman Pichler](https://twitter.com/johncutlefish)
+
+## Product L & D Hub への貢献方法
+
+Product L & D Hub を堅牢で関連性のあるものに保つのは、チームの努力です。役に立つリソースを見つけたら貢献することをお勧めします。Product L & D Hub に古くなった、または同意できないリソースがあれば、それも私たちに知らせることを推奨します。
+
+Product L & D ハブには 2 つのコンポーネントがあります。
+
+このライブラリに追加または削除したいコンテンツがある場合は、MR を作成し、以下にリストされている [product learning evangelists](#product-learning-evangelists) を `Reviewer` として追加してください。コンテンツについて何を評価しているか、どのように役立つと考えているかの詳細を MR の説明に追加する必要があります。MR に `milestone` とラベル `product operations`、`product handbook`、`prodops:release` を追加してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)